templates/students/attendance.html

```diff
@@ -6,145 +6,198 @@
-                            {% else %}
-                                <td>{{stu_att.student.school_class}}</td>
-                            {% endif %}
-
-                            <td>
-                                <div class="custom-control custom-switch">
-                                    <input type="checkbox" name="attended" value="{{stu_att.student.id}}" class="custom-control-input"
-                                        id="s{{ stu_att.id }}" {% if stu_att.present %}checked{% endif %}>
-                                    <label class="custom-control-label" for="s{{ stu_att.id }}"></label>
-                                </div>
-                            </td>
-                        </tr>
-                        {% endfor %}
-                    </tbody>
-                </table>
-            </div>
-        </div>
-        <!-- <p class="text-center text-danger mt-1">{{cwhw_error}}</p> -->
-        <div class="row justify-content-center mb-4">
-            <div class="col-md-4">
-                <button name="submit" value="stu-att" class="btn btn-primary btn-lg btn-block" type="submit">Submit</button>
-            </div>
-        </div>
-    </form>
-    {% endif %}
-</div>
+                <!-- <p class="text-center text-danger mt-1">{{cwhw_error}}</p> -->
+                <div class="row justify-content-center mb-4">
+                    <div class="col-md-4">
+                        <button name="submit" value="stu-att" class="btn btn-primary btn-lg btn-block" type="submit">
+                            Submit
+                        </button>
+                    </div>
+                </div>
+            </form>
+        {% endif %}
+    </div>
 
 {% endblock %}
 
 {% block scripts-sidebar %}
 
-<!-- AJAX for Student Attendance -->
-
-<script>
-    // $("#stu-att-class option:selected").removeAttr('selected');
-    $("#stu-att-class option:first").attr('selected', 'selected');
-    $("#stu-att-class").change(function () {
-        let stu_class = $(this).val();
-
-        $.ajax({
-            url: "{% url 'students:ajax_attendance' %}",
-            data: {
-                'stu_class': stu_class,
-            },
-            dataType: 'json',
-            success: function (data) {
-                let stu_att_table = '<thead><tr><th>Student Name</th><th>Class</th><th>Mark Present</th></tr></thead><tbody>';
-                let checked = '';
-
-                Object.entries(data).forEach(([key, [stu_id, name, school_class, present]]) => {
-                    if (present) checked = 'checked';
-                    else checked = '';
-
-                    if(school_class == 13) school_class = "JNV_4"
-                    else if(school_class == 14) school_class = "JNV_5"
-
-                    stu_att_table += '<tr><td>' + name + '</td><td>' + school_class + '</td> \
+    <script>
+        function attendance(student_id, checkboxID) {
+            let student = document.getElementById('s' + checkboxID);
```

Comment:
Variable name `student` doesn't seem correct here. You should use a more descriptive name like `att_checkbox` instead.